tests/testthat.R

```diff
@@ -0,0 +1,12 @@
+# This file is part of the standard setup for testthat.
+# It is recommended that you do not modify it.
+#
+# Where should you do additional test configuration?
+# Learn more about the roles of various files in:
+# * https://r-pkgs.org/testing-design.html#sec-tests-files-overview
+# * https://testthat.r-lib.org/articles/special-files.html
+
+library(testthat)
+library(RWEDataManagementModel)
+
+test_check("RWEDataManagementModel")
```

Comment:
I ran this script but there was an error. Indeed test_dir() is not found in the package
Error in `test_dir()`:
! No test files found

Reply:
Hi @YMao-UMCU, thanks for your comments! Shall we go through the review together?